recipes/hpsklearn/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "hpsklearn" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+# https://github.com/hyperopt/hyperopt-sklearn/issues/202
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hpsklearn-{{ version }}.tar.gz
+    sha256: 3a320062eb3296b35cccac87ad8b28b95dbc5f1fad22dc81e911f26d5b491cb7
+  - url: https://github.com/hyperopt/hyperopt-sklearn/archive/840cfc9a71d26b08a59b22b5bd83072873f44918.tar.gz
+    sha256: 08f5f11d4026387ca25e6642a19f56f6d62cb34a9b77f12ee9588c75c28034be
+    folder: github-source
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - hyperopt
+    - nose
```

Comment:
Is nose really needed at run time?  Maybe it is just a test dependency leaking to the runtime.

```suggestion
```